wrappers/src/shared_realm_cs.cpp

```diff
@@ -547,4 +548,38 @@ REALM_EXPORT Results* shared_realm_create_results(SharedRealm& realm, TableKey t
     });
 }
 
+REALM_EXPORT void shared_realm_rename_property(const SharedRealm& realm, uint16_t* type_name_buf, size_t type_name_len,
+    uint16_t* old_name_buf, size_t old_Name_len, uint16_t* new_name_buf, size_t new_name_len, Schema* migration_schema, NativeException::Marshallable& ex)
+{
+    return handle_errors(ex, [&]() {
+        Utf16StringAccessor type_name_str(type_name_buf, type_name_len);
+        Utf16StringAccessor old_name_str(old_name_buf, old_Name_len);
+        Utf16StringAccessor new_name_str(new_name_buf, new_name_len);
+
+        ObjectStore::rename_property(realm->read_group(), *migration_schema, type_name_str, old_name_str, new_name_str);
+    });
+}
+
+REALM_EXPORT bool shared_realm_remove_type(const SharedRealm& realm, uint16_t* type_name_buf, size_t type_name_len, NativeException::Marshallable& ex)
+{
+    return handle_errors(ex, [&]() {
+        Utf16StringAccessor type_name_str(type_name_buf, type_name_len);
+
+        auto table = ObjectStore::table_for_object_type(realm->read_group(), type_name_str);
+        if (!table)
+        {
+            return false;
```

Comment:
Isn't this a similar problem to the one below? Should we throw here as well?

Reply:
Yeah, this is up for discussion I think 😁 
The one down throws an exception because it tries to do something that can be already done with the current API (`realm.RemoveAll`), and I suppose it's better to have developers having one way of doing certain things, as it causes less headaches

Reply:
Hm, if this is a valid way, just not the preferred one I would rather add a comment to the documentation than throwing a  (probably) very unexpected and confusing error because it's supposed to work? I get your point, just trying to see it from a users point of view.